clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -533,6 +535,25 @@ public void testMetadataChangeTriggersRebalance() {
     }
 
     @Test
+    public void testExcludeInternalTopicsConfigOption() { 
```

Comment:
@gwenshap in the unit test there are no connections to the server and different unit tests in that class do not interfere with each other (as it should be :-) so I think the catch-all pattern is fine.